src/openrct2/paint/tile_element/Paint.Path.cpp

```diff
@@ -447,17 +447,20 @@ static void sub_6A4101(
             uint16_t scrollingMode = railingEntry->scrolling_mode;
             scrollingMode += direction;
 
-            set_format_arg(0, uint32_t, 0);
-            set_format_arg(4, uint32_t, 0);
+            // FIXME this is a bit hackish. We clear the 8 first bytes just in case
+            // we don't write to all of them later on. Check if really necessary.
+            Formatter::Common().Add<uint32_t>(0).Add<uint32_t>(0);
```

Comment:
It is required due to `scrolling_text_setup` doing a memcmp of another set of args to see if it matches this set of args.
Eventually when `scrolling_text_setup` takes the formatter variable this can be removed.

Reply:
I updated the comment to describe that so the next developer has this hint. Hopefully not many developers will pass here. I hope to get these things cleaned up. It will be beautiful :)